src/test/resources/edu/ie3/datamodel/io/sink/_sql/load_profile.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE public.load_profile_g2
```

Comment:
What is this _g2 supposed to mean?

Reply:
It's the standard load profile with key "g2"